src/manage.c

```diff
@@ -678,6 +678,18 @@ static void manage_tracking_startup(void)
       continue;
     }
 
+    /* Transition to tracking. */
```

Comment:
Remove this. The u64 timing count stuff is unnecessary with the measurement time delta changes.